local_groups/models.py

```diff
@@ -10,37 +10,37 @@
 from django.contrib.gis.db.models import PointField
 
 
-class Group(models.Model):    
```

Comment:
not sure why getting so many whitespace changes. something to do with atom maybe.

Reply:
No whitespace on empty lines is preferred, so your editor is correct here. Someone might have been using improperly configured editor before.

Reply:
Ok cool 👍